### PR TITLE
Remove old `2DII` document link from the `README.md`

### PR DESCRIPTION
**Pull request by jdhoffa**
This document is largely superseded and no longer used. 

I would suggest that any discussions occur either in the issues or PRs of this repository (or also verbally in Dev Hangouts and or Tech Reviews, in which case I will do my best to document and paraphrase the discussions). 

(I also .gitignore the Rproj, as this is not relevant to this repo)

@AlexAxthelm I don't think the whole team needs to review this, it's mostly just tidy up of the files/ removing files we no longer use...

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .Rproj.user
 .Rhistory
 git.html
+practices.Rproj
```
Comment (cjyetman) [inline .gitignore:4]: sneaking unrelated stuff into a PR... is there an issue for that? 🤣

**Review by cjyetman — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (jdhoffa) [reply .gitignore:4]: LOL, as I was doing, I was like "man i shouldn't do this"

Comment (jdhoffa) [reply .gitignore:4]: but oh well, we don't have any written practices so I think I can still do whatever I want 😂
